packages/installations/src/helpers/fid-changed.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getKey } from '../util/get-key';
+import { AppConfig } from '../interfaces/app-config';
+import { IdChangeCallbackFn } from '../functions';
+
+const FID_CHANGE_CALLBACKS: Map<string, Set<IdChangeCallbackFn>> = new Map();
```

Comment:
"Constants: CONSTANT_CASE indicates that a value is intended to not be changed, and may be used for values that can technically be modified (i.e. values that are not deeply frozen) to indicate to users that they must not be modified."

Pretty sure you modify this :p

Reply:
Done.